client/src/js/directives/bhSubmit.js

```diff
@@ -0,0 +1,40 @@
+angular.module('bhima.directives')
+.directive('bhSubmit', bhSubmitDirective);
+
+function bhSubmitDirective() {
+  return {
+    restrict : 'A',
+    require : 'form', // make sure we are on an ngForm element
+    priority : 1,     // This must be a number greater than zero
+    scope : {
+      submit: '&bhSubmit'
+    },
+    link : function bhSubmitLinkFn($scope, $element, $attrs, $controller) {
```

Comment:
Angular injects it by default, there is no way to get rid of it, to my knowledge.